gui/wxpython/lmgr/workspace.py

```diff
@@ -0,0 +1,463 @@
+"""
+@package lmgr::workspace
+
+@brief Workspace manager class for creating, loading and saving workspaces
+
+Class:
+ - lmgr::WorkspaceManager
+
+(C) 2021 by the GRASS Development Team
+
+This program is free software under the GNU General Public
+License (>=v2). Read the file COPYING that comes with GRASS
+for details.
+"""
+
+import sys
+import os
+import tempfile
+
+try:
+    import xml.etree.ElementTree as etree
+except ImportError:
+    import elementtree.ElementTree as etree  # Python <= 2.4
```

Comment:
Check the correct import for Py3 and remove, we don't keep compatibility with Python2 anymore.